apps/payments/next/next.config.js

```diff
@@ -48,6 +49,7 @@ const nextConfig = {
       '@sentry/nestjs',
       '@sentry/open-telemetry',
       '@type-cacheable/core',
+      'app-store-server-api',
```

Comment:
Does the google package I added last week need to be added here as well?

Reply:
Ah yes definitely should. I'll make the change.